mobile/src/profile/chevron.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, LayoutChangeEvent } from 'react-native';
+import { layoutStyles } from './styles';
+
+interface ChevronProps {
+    containerWidth?: number;
+}
+
+interface ChevronState {
+    chevronWidth: number;
+}
+
+export class Chevron extends Component<ChevronProps, ChevronState> {
```

Comment:
Don't we have such functionality already? From calendar. We just need to come up with a good name for this `common` folder as I HATE 'common' or 'shared' words. styling, design-components, etc, suggestions are welcome?

Reply:
or just somewhere in layout folder.

Reply:
I've thought that there would be no chevrons in future releases of calendar...
And this one is better - it doesn't require any external dimensions

Reply:
you might be right. Let's catch Ilya one more time today